advanced_source/dynamic_quantization_tutorial.py

```diff
@@ -98,9 +98,9 @@ def __len__(self):
 class Corpus(object):
     def __init__(self, path):
         self.dictionary = Dictionary()
-        self.train = self.tokenize(os.path.join(path, 'train.txt'))
-        self.valid = self.tokenize(os.path.join(path, 'valid.txt'))
-        self.test = self.tokenize(os.path.join(path, 'test.txt'))
+        self.train = self.tokenize(os.path.join(path, 'wiki.train.token'))
+        self.valid = self.tokenize(os.path.join(path, 'wiki.valid.token'))
+        self.test = self.tokenize(os.path.join(path, 'wiki.test.token'))
```

Comment:
Why are you doing those changes? Where those files are supposed to be coming from?

Reply:
These were the paths when I downloaded the dataset myself. Without these changes I was not able to run the tutorials. I guess CI downloads the files differently somehow

Reply:
To unblock this PR maybe we should just revert these changes and investigate separately. How does that sound @jcaip?

Reply:
Hmm, downloaded from where? For tutorials build, download locations are defined in (plan to move them to json file, but this is where they are for now)
https://github.com/pytorch/tutorials/blob/4918fe4bf3df96b233ca134c985ffb09ea45f92c/Makefile#L93-L94

Reply:
I was using a different wikitext version which has changed the naming convention of those files, which is causing the error. I will update the internal tutorial guide and revert the filepaths, which I think should fix everything. Thanks for the help @malfet!